test/test_examples.sh

```diff
@@ -5,53 +5,63 @@
 
 set -e
 
-WORKSPACE=$1
+SOURCE_DIR=$1
 BUILD_DIR=$2
 INSTALL_DIR=$3
+CMAKE_INSTALL_PREFIX=$4
```

Comment:
I'm not sure - why do we have that prefix here? we already have install_dir, which is set for this test to be inside cmake_current_binary_dir

Reply:
Please read carefully https://github.com/oneapi-src/unified-memory-framework/pull/679#discussion_r1717288909 (August 14th):

The command:
```
$ make DESTDIR=/package/stage install
```
installs all files to the `${DESTDIR}/${CMAKE_INSTALL_PREFIX}` directory.